data_structures/src/lib.rs

```diff
@@ -2,7 +2,6 @@
 // To enable `#[allow(clippy::all)]`
 //#![feature(tool_lints)]
 
-
```

Comment:
Remove data_structures/ from the commit

Reply:
this diff is generated by rustfmt, the code should be formatted as listed in `Justfile` taskes.

Reply:
True, but you should put it in a separated commit: `refactor(data_structures): fix format`